entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+set -u
+
+BUILD=/build
+WORKD=/data
+
+build_working_dir() {
+    [ -f $WORKD/nitter.conf ] || cp -f  $BUILD/nitter.conf $WORKD/.
+    [ -d $WORKD/public ]      || cp -rf $BUILD/public      $WORKD/.
+    [ -d $WORKD/tmp ]         || mkdir -p $WOKRD/tmp
```

Comment:
Typo
```suggestion
    [ -d $WORKD/tmp ]         || mkdir -p $WORKD/tmp
```

Reply:
Good catch!  I will make the fix and generate a new merge request.

Reply:
No need for a new request, just push the changes to your branch

Reply:
OK.  Done!